pkg/rancher-desktop/config/settings.ts

```diff
@@ -427,6 +429,7 @@ const updateTable: Record<number, (settings: any) => void> = {
       memoryInGB:   settings.kubernetes.memoryInGB,
       numberCPUs:   settings.kubernetes.numberCPUs,
     };
+    settings.experimental = { virtualMachine: { socketVMNet: settings.kubernetes.socketVMNet } };
```

Comment:
5 lines up we're setting `settings.virtualMachine.experimental`; that should go away.

Reply:
Thanks, removed.

Also `settings.kubernetes.socketVMNet` should have beeen `settings.kubernetes.experimental.socketVMNet`. I fixed that as well.